core/cap-0028.md

```diff
@@ -11,7 +11,7 @@ Protocol version: 13
 ```
 
 ## Simple Summary
-Pre-auth signers are only removed from the source account if signature verification on the transaction succeeds. The signer is left on the account otherwise. This proposal will remove the signer from the source account when the transaction is being applied, regardless of the outcome of that transaction.
+Pre-auth signers are only removed from the source account if signature verification on the transaction succeeds during transaction application. The signer is left on the account otherwise. This proposal will remove the signer from the source account when the transaction is being applied, regardless of the outcome of that transaction.
```

Comment:
Is there a definition of what _transaction application_ means, formally? I think it should be referenced.

Reply:
_Transaction application_ is just the step where the transaction is executed to modify the ledger, as opposed to the earlier step where the transaction is validated and included in the transaction set, but not applied yet. I added a second commit that hopefully adds enough information. Please let me know what you think.